frontend/cypress/e2e/retornaparalista.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Retorna à lista de clientes ao clicar no botão "Voltar" do Customer Details', () => {
+    it('deve retornar à lista de clientes ao clicar no botão "Voltar"', () => {
+        cy.visit('http://localhost:3000');
+
+        cy.get('tbody tr').first().find('button[aria-label^="View company"]').click();
```

Comment:
Veja uma solução mais simples:

```suggestion
        cy.contains('button', 'View').click();
```